.gitignore

```diff
@@ -6,3 +6,4 @@ components/*/test/screenshots/golden/
 mixins/*/test/screenshots/current/
 mixins/*/test/screenshots/golden/
 test/sass.output.css
+.DS_Store
```

Comment:
Sorry; this is not strictly related. Viewing the golden images in Finder in MacOS dirties the repo with these files.

Reply:
I just have a global ignore rule locally on my machine... otherwise we end up with `.DS_Store`, plus Visual Studio user files, plus Atom user files etc. etc.